pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -213,3 +214,20 @@ func SeverityFromPriority(priority string) types.Severity {
 		return types.SeverityUnknown
 	}
 }
+
+// StatusFromUbuntuStatus normalises Ubuntu status into common Trivy Types
+func StatusFromUbuntuStatus(status string) types.Status {
+	switch status {
+	case "not-affected", "DNE", "not-vulnerable", "ignored":
+		return types.StatusWillNotFix
+	case "deffered":
+		return types.StatusFixDeferred
+	case "needed", "pending":
+		return types.StatusPending
```

Comment:
```suggestion
	case "deferred", "needed", "pending":
		return types.StatusFixDeferred
```
Corrected what appears to be a typo in the `case`, and suggesting these are treated as the same case to avoid needing to extend the supported state list for Trivy DB.

Reply:
Good catch on the typo

I have compressed StatusPending into StatusFixDeferred